src/AWS.Deploy.Orchestration/DisplayedResources/S3BucketResource.cs

```diff
@@ -20,26 +20,30 @@ public S3BucketResource(IAWSResourceQueryer awsResourceQueryer)
 
         public async Task<Dictionary<string, string>> Execute(string resourceId)
         {
-            string region = await _awsResourceQueryer.GetS3BucketLocation(resourceId);
-            string regionSeparator = ".";
-            if (string.Equals("us-east-1", region) ||
-                string.Equals("us-west-1", region) ||
-                string.Equals("us-west-2", region) ||
-                string.Equals("ap-southeast-1", region) ||
-                string.Equals("ap-southeast-2", region) ||
-                string.Equals("ap-northeast-1", region) ||
-                string.Equals("eu-west-1", region) ||
-                string.Equals("sa-east-1", region))
+            var metadata = new Dictionary<string, string> { { "Bucket Name", resourceId } };
+
+            var webSiteConfiguration = await _awsResourceQueryer.GetS3BucketWebSiteConfiguration(resourceId);
+            if(webSiteConfiguration != null && !string.IsNullOrEmpty(webSiteConfiguration.IndexDocumentSuffix))
```

Comment:
This change to the S3 Bucket displayed resources is to only added the EndpointURL if the S3 bucket has a website configured.

Reply:
Done